miio/__init__.py

```diff
@@ -29,7 +29,7 @@
 from miio.airhumidifier_mjjsq import AirHumidifierMjjsq
 from miio.airpurifier import AirPurifier
 from miio.airpurifier_airdog import AirDogX3, AirDogX5, AirDogX7SM
-from miio.airpurifier_miot import AirPurifierMB4, AirPurifierMiot
+from miio.airpurifier_miot import AirPurifierMiot
```

Comment:
Let's keep AirPurifierMB4 as a proxy with deprecated warning for a while, just in case someone is using that.

Please add a class to `miio.airpurifier_miot` module, something like this:
```
from miio.utils import deprecated

class AirPurifierMB4(AirPurifierMiot):
    @deprecated("Use AirPurifierMiot")
    def __init__(*args, **kwargs):
        super().__init__(*args, **kwargs)
```